client/components/info-popover/README.md

```diff
@@ -57,25 +57,26 @@ that can trigger the opening and closing of the InfoPopover then you need to pas
 ```
 
 ```js
-const refs = {};
+const infoPopRef = React.createRef();
```

Comment:
The same thing as above for the `const` here. Usually we define those as class instance properties, like this:

```
class ExampleComponent extends Component {
	buttonRef = React.createRef();

	render() {
		return (
			<Button ref={ this.buttonRef }>Something</Button>
		);
	}
}
```